content/installation/python/PythonSupportedTechnologies.md

```diff
@@ -1,46 +1,70 @@
-<!-- 
+<!--
 title: "Supported Technologies"
 description: "List of supported technologies"
 tags: "installation Python agent frameworks support package"
 -->
 
-The Python agent supports Python versions 2.7 and 3.5 to 3.8. Framework support is currently for:
+## Contrast agents
```

Comment:
I find the generality / generalization? here confusing since we are within docs of Contrast Agents. What about "agent version support"?

Reply:
FYI, this section header will not appear in the new documentation. Headers will read: "Language version" and "Application frameworks"

Reply:
Given that it's going away, I'm going to leave as is for now so that it matches other pages.